frontend/ui/uimanager.lua

```diff
@@ -876,7 +876,15 @@ function UIManager:sendEvent(event)
         end
     end
 
+    -- The top widget get to be the first to get the event
     local top_widget = self._window_stack[#self._window_stack]
+    if top_widget.widget.toast then
```

Comment:
I guess this should be a while top_widget.widget.toast  - so we dequeue all .toast that could be stacked (but we'll stop at a modal non .toast)

Reply:
Good point!

(Might happen for real with stuff batched by Dispatcher?)